debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ContinuationStackWalkCommand.java

```diff
@@ -0,0 +1,105 @@
+ * Copyright IBM Corp. and others 2023
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+package com.ibm.j9ddr.vm29.tools.ddrinteractive.commands;
+
+import java.io.PrintStream;
+
+import com.ibm.j9ddr.CorruptDataException;
+import com.ibm.j9ddr.tools.ddrinteractive.Command;
+import com.ibm.j9ddr.tools.ddrinteractive.CommandUtils;
+import com.ibm.j9ddr.tools.ddrinteractive.Context;
+import com.ibm.j9ddr.tools.ddrinteractive.DDRInteractiveCommandException;
+import com.ibm.j9ddr.vm29.j9.DataType;
+import com.ibm.j9ddr.vm29.j9.stackwalker.BaseStackWalkerCallbacks;
+import com.ibm.j9ddr.vm29.j9.stackwalker.StackWalkResult;
+import com.ibm.j9ddr.vm29.j9.stackwalker.StackWalker;
+import com.ibm.j9ddr.vm29.j9.stackwalker.StackWalkerUtils;
+import com.ibm.j9ddr.vm29.j9.stackwalker.TerseStackWalkerCallbacks;
+import com.ibm.j9ddr.vm29.j9.stackwalker.WalkState;
+import com.ibm.j9ddr.vm29.pointer.generated.J9BuildFlags;
+import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
+import com.ibm.j9ddr.vm29.pointer.helper.J9RASHelper;
+import com.ibm.j9ddr.vm29.structure.J9Consts;
+import com.ibm.j9ddr.vm29.tools.ddrinteractive.JavaVersionHelper;
+
+/**
+ * DDR command to walk the stack frames of a continuation.
+ */
+public class ContinuationStackWalkCommand extends Command {
+	public ContinuationStackWalkCommand() {
+		addCommand("continuationstack", "<continuation>", "Walks the Java stack for <continuation>");
+		addCommand("continuationstackslots", "<continuation>", "Walks the Java stack (including objects) for <continuation>");
+	}
+
+	@Override
+	public void run(String command, String[] args, Context context, PrintStream out) throws DDRInteractiveCommandException {
+		try {
+			final J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
+
+			if (!JavaVersionHelper.ensureMinimumJavaVersion(19, vm, out)) {
+				CommandUtils.dbgPrint(out, "Continuations are only available in Java 19 or newer.\n");
```

Comment:
Suggest removing this message as it is redundant.
```
> !continuationstack 0x00154100
This command only works with core file created by VM with Java version 19 or higher.
The current VM Java version is: 17.
Continuations are only available in Java 19 or newer.
```

Reply:
I suggest removing the version check entirely. DDR is a debug tool - you're supposed to know what you're doing. Running the command on an old core and getting CorruptDataException, or missing class/field messages is perfectly acceptable.

Reply:
There's a significant number of places where we test the version of the VM that produced the dump (mostly regarding modularity), so keeping this test seems consistent to me. I'd just like the redundant part of the feedback removed.

Reply:
Removed.